st2client/st2client/shell.py

```diff
@@ -65,7 +65,7 @@
 from st2client.utils.logging import LogLevelFilter, set_log_level_for_all_loggers
 from st2client.commands.auth import TokenCreateCommand
 from st2client.commands.auth import LoginCommand
-
+from st2client.utils.profiler import setup_regular_profiler
```

Comment:
NOTE: st2client can't depend on st2common that's why we intentionally need to duplicate the code in this package.